### DISCUSSION TITLE
Github Actions Storage Calculation

### DISCUSSION THREAD

**Discussion by dLamSlo8** [community/community#26667] [Actions]
<p>Hi all, I was wondering what counts towards storage usage for GitHub Actions? I’m currently using it to optimize images, which creates new images and commits them to the same branch. I’m not seeing any storage usage when doing that in my Billing settings, so I’m curious what actually counts towards calculating storage use? Thank you!</p>

**Comment by Akash1134** [comment, marked as answer]
<p>Hi <a class="mention" href="/dlamslo8">@dLamSlo8</a> ,</p>
<p>GitHub Actions usage is free for both public repositories and self-hosted runners. For private repositories, each GitHub account receives a certain amount of free minutes and storage, depending on the product used with the account. Any usage beyond the included amounts is controlled by spending limits.</p>
<p>All artifacts in private repositories are counted towards your storage. For more details, please check out: <a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/managing-your-spending-limit-for-github-actions">Managing your spending limit for GitHub Actions</a></p>

**Reply by lildevlady** [reply]
I have removed a lot of artifacts and expected after more than 12 hours to have the ability to run actions again, but it still fails and I don't understand why?

**Comment by dLamSlo8** [comment]
<p>Thank you! I had to look up what an artifact was, but here’s a link for those who also don’t know: <a href="https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts" rel="noopener nofollow ugc">Storing workflow data as artifacts - GitHub Docs</a></p>
